o-fish-ios/Views/LoginView.swift

```diff
@@ -45,12 +45,10 @@ struct LoginView: View {
                                     text: self.$username)
                                     .keyboardType(.emailAddress)
 
-                                //TODO need to implement Face id
-                                Button(action: { print("Face id") }) {
-                                    Image(systemName: "faceid")
+                                Button(action: self.getStoredCredentials) {
+                                    Image(systemName: KeychainWrapper.shared.getPictureName())
```

Comment:
lets inject this inside the view struct:

```
struct LoginView {
...
   private let keychain = KeychainWrapper.shared
....
```

and use as a 
`self.keychain.getPictureName()` or `keychain.removeCredentials()`, etc